volontulo/tests/views/test_usersprofile.py

```diff
@@ -0,0 +1,224 @@
+# -*- coding: utf-8 -*-
+
+u"""
+.. module:: test_users
+"""
+from django.contrib.auth.models import User
+from django.test import Client
+from django.test import TestCase
+
+from volontulo.models import Badge
+from volontulo.models import Offer
+from volontulo.models import Organization
+from volontulo.models import UserBadges
+from volontulo.models import UserProfile
+
+
+class TestUsersProfile(TestCase):
+    u"""Class responsible for testing offers' list."""
+
+    @classmethod
+    def setUpTestData(cls):
+        # volunteer user - totally useless
+        cls.initialize_empty_volunteer()
+        # organization user - no offers
+        cls.initialize_empty_organization()
+        # volunteer user - badges, offers, organizations
+        cls.initialize_filled_volunteer_and_organization()
+
+    @classmethod
+    def initialize_empty_volunteer(cls):
+        u"""Initialize empty volunteer."""
+        volunteer_user1 = User.objects.create_user(
+            'volunteer1@example.com',
+            'volunteer1@example.com',
+            'volunteer1'
+        )
+        volunteer_user1.save()
+        cls.user_profile1 = UserProfile.objects.create(user=volunteer_user1)
+
+    @classmethod
+    def initialize_empty_organization(cls):
+        u"""Initialize empty organization."""
+        organization1 = Organization.objects.create(name=u'Organization 1')
+        organization1.save()
+        organization_user1 = User.objects.create_user(
+            'organization1@example.com',
+            'organization1@example.com',
+            'organization1'
+        )
+        organization_user1.save()
+        cls.organization_profile1 = UserProfile.objects.create(
+            user=organization_user1,
+        )
+        cls.organization_profile1.organizations.add(organization1)
+
+    @classmethod
+    # pylint: disable=invalid-name
+    def initialize_filled_volunteer_and_organization(cls):
```

Comment:
It doesn't look consistent if You have 2 separate methods for creation of empty volunteer and organization and only 1 method for filled volunteer and organization.